Documentation/D2.md

```diff
@@ -21,17 +21,30 @@ During our first meeting we discovered that:
 
 ## Metrics:
 We plan on evaluating the following metrics in order:
-* Response Time (Time between local commit, remote push, or pull request and recorded output event from the RPi code)
+* Response Time
+  * Experiment: We will be measuring the response time of the device from the time the user preforms a valid action to the time the device is triggered to respond. Valid actions are: local commits, remote pushes, or pull requests. The time the device sends the signal to activate the connected devices.
+  * Procedure: We will be preforming a retroactive analysis of the events that occure. Step one is to preform 30 valid actions. We will then pull the github logs timestamps and the RPis logs timestamps and compute the average difference to determine action laitency.
```

Comment:
Step one is to push 10 times to a remote branch. We will then record the time the commit was received in the local branch and compare it to the time when then fan motor was turned on.